packages/create-react-admin/templates/common/package.json

```diff
@@ -1,5 +1,6 @@
 {
     "private": true,
+    "type": "module",
```

Comment:
that's a strong choice. If it forces users to use cjs instead of js, I'm -1 for it.

Reply:
This is the default for vite templates with TS/React and thanks to the TS config it just work (`moduleResolution=Bundler)